aptos-move/move-examples/resource_groups/secondary/sources/secondary.move

```diff
@@ -0,0 +1,58 @@
+/// This demonstrates how to use a resource group accross modules
+module resource_groups_secondary::secondary {
+    use std::signer;
+    use resource_groups_primary::primary;
+
+    #[resource_group(container = resource_groups_primary::primary::ResourceGroupContainer)]
```

Comment:
from my understanding, the random modules would need to to have the signer of the resource group address, as putting something into the resource group uses `move_to`

Reply:
we added scope to the proposal to mitigate these concerns.

Reply:
I feel like if you have the signer already, there is much more malicious action that you could do than just stick a bunch of random data in there, which should also be expensive to do: or is there a hypothetical attack I'm missing?

Reply:
Resource group is by account so if a module has a signer for account 0x123, it can add to the group for that account. This would penalize the performance for that account for other resources/modules in the same resource group. This might not be from malicious intention but would increase the costs for everyone.